src/Validator/Validator.php

```diff
@@ -2,10 +2,13 @@
 
 namespace RicardoFiorani\Validator;
 
-use GuzzleHttp\Psr7\Request;
 use Psr\Http\Client\ClientInterface;
+use Psr\Http\Message\RequestFactoryInterface;
 use RicardoFiorani\Validator\Response\ValidationResponseFactory;
 use RicardoFiorani\Validator\Response\ValidationResponseInterface;
+use Zend\Diactoros\RequestFactory;
+
+use function GuzzleHttp\Psr7\stream_for;
```

Comment:
Guzzle is a dev dependency only, it won't work on a real integration.

Instead of relying on Guzzle, you can use the diactoros stream factory that follows PSR-17 suggestion for creating streams : https://www.php-fig.org/psr/psr-17/#24-streamfactoryinterface
E.g.:
```
// Namespace as http-interop/http-factory-diactoros
(new \Http\Factory\Diactoros\StreamFactory())->createStream($content);
```

Maybe we could send the stream factory as an optional argument as well ? But this is optional, I'm suggesting doing this but I might create another issue for it.

Reply:
I think it's ok without an optional parameters.